command-parser.js

```diff
@@ -634,7 +635,8 @@ let parse = exports.parse = function (message, room, user, connection, levelsDee
 	});
 
 	if (commandHandler) {
-		return context.run(commandHandler);
+		message = context.run(commandHandler);
+		if (!message || typeof message.then === 'function') return message;
```

Comment:
It's still used for testing asynchronous commands. The build is currently failing because of the <code>/learn</code> tests.